sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/command/UpdateGatewayRuleCommandHandler.java

```diff
@@ -52,10 +54,42 @@
         RecordLog.info(String.format("[API Server] Receiving rule change (type: gateway rule): %s", data));
 
         String result = SUCCESS_MSG;
-        List<GatewayFlowRule> flowRules = JSONArray.parseArray(data, GatewayFlowRule.class);
-        GatewayRuleManager.loadRules(new HashSet<>(flowRules));
+        Set<GatewayFlowRule> flowRules = JSON.parseObject(data,new TypeReference<Set<GatewayFlowRule>>(){});
```

Comment:
Could you please reformat your code?

Reply:
@zhangyide One space after comma makes code better reading, using IDE to format this line is okay : )

Reply:
ok. I had reformated it.